Example/ViewController.swift

```diff
@@ -260,7 +260,7 @@ class ViewController: UIViewController {
     func startBasicNavigation() {
         guard let response = response, let route = response.routes?.first, case let .route(routeOptions) = response.options else { return }
         
-        let service = navigationService(route: route, options: routeOptions)
+        let service = navigationService(route: route, routeIndex: 0, options: routeOptions)
```

Comment:
In many cases, the developer will be able to hard-code 0 as the route index after getting the `first` route out of the route array. But I made the argument required because I didn’t want any code that deals with multiple routes to go unnoticed during an upgrade.